gui/app/controls/ParameterAdjustPopup.qml

```diff
@@ -0,0 +1,112 @@
+import QtQuick 2.11
+import QtQuick.Layouts 1.3
+import QtQuick.Window 2.2
+import QtQuick.Controls 2.4
+
+import ".."
+
+/*!
+    \qmltype ParameterAdjustPopup
+    \brief A popup control that allows user to change parameter values
+*/
+BasePopup {
+    id: root
+
+    // informational text about value displayed
+    property string parameterInfoText
+
+    // unit text about value displayed
+    property string parameterUnit
+
+    // current value for the paramater
+    property real value: 0
+
+    // minimum value for the parameter
+    property real minValue: 0
+
+    // maximum value for the parameter
+    property real maxValue: 512
+
+    // step value used for incrementing or decrementing value
+    property real stepSize: 1
+
+    // formatter function used to display parameter value
+    property var displayFormatter: function(value) {
+        return value.toString()
+    }
+
+    signal valueConfirmed
+
+    onConfirm: root.valueConfirmed()
+
+    Item {
+
+        Text {
+            id: parameterInfo
+            anchors {
+                horizontalCenter: parent.horizontalCenter
+                top: parent.top; topMargin: 46
+            }
+            text: root.parameterInfoText
+            font: Style.theme.font.textHeaderSubtitle
+            color: Style.theme.color.textPrimary
+        }
+
+        BaseButton {
+            id: decreaseButton
+            anchors {
+                left: parent.left; leftMargin: 176
+                top: parent.top; topMargin: 168
+            }
+
+            contentItem: Image {
+                fillMode: Image.PreserveAspectFit
+                source: 'qrc:/images/Logo.png'
```

Comment:
Can we change decrease and increase buttons to just say "+" and "-" rather than having the PV logo in them?

Reply:
I updated with the just arrived logos